### PR TITLE
2 packages from puripuri2100/satysfi-class-mdbook-satysfi at 0.5.1

### PR DESCRIPTION
**Pull request by puripuri2100**
A SATySFi class file for processing document file generated by mdbook-satysfi

This pull-request concerns:

- `satysfi-class-mdbook-satysfi.0.5.0`
- `satysfi-class-mdbook-satysfi-doc.0.5.0`

---

- Homepage: https://github.com/puripuri2100/satysfi-class-mdbook-satysfi
- Source repo: git+https://github.com/puripuri2100/satysfi-class-mdbook-satysfi.git
- Bug tracker: https://github.com/puripuri2100/satysfi-class-mdbook-satysfi/issues
# Automatic follow-ups
Choose follow-up actions.  Do not write anything after this section.
- [x] Add to snapshot `snapshot-develop` :: satysfi-class-mdbook-satysfi-doc.0.5.1 satysfi-class-mdbook-satysfi.0.5.1
- ~~Add to snapshot `snapshot-stable-0-0-4`~~ (Inconsistent)
- ~~Add to snapshot `snapshot-stable-0-0-5`~~ (Inconsistent)
- ~~Add to snapshot `snapshot-stable-0-0-6`~~ (Inconsistent)
- ~~Add to snapshot `snapshot-stable-0-0-6--1`~~ (Inconsistent)
- [x] Add to snapshot `snapshot-stable-0-0-7` :: satysfi-class-mdbook-satysfi-doc.0.5.1 satysfi-class-mdbook-satysfi.0.5.1
- [x] Add to snapshot `snapshot-stable-0-0-8` :: satysfi-class-mdbook-satysfi-doc.0.5.1 satysfi-class-mdbook-satysfi.0.5.1

### CODE REVIEW

File: packages/satysfi-class-mdbook-satysfi-doc/satysfi-class-mdbook-satysfi.0.5.0/opam [hunk 1]
```diff
@@ -0,0 +1,41 @@
+opam-version: "2.0"
```
Comment (na4zagin3) [inline packages/satysfi-class-mdbook-satysfi-doc/satysfi-class-mdbook-satysfi.0.5.0/opam:1]: Can you fix the directory name?

```
packages/satysfi-class-mdbook-satysfi-doc/satysfi-class-mdbook-satysfi.0.5.0/opam

to

packages/satysfi-class-mdbook-satysfi-doc/satysfi-class-mdbook-satysfi-doc.0.5.0/opam
```

Comment (na4zagin3) [inline packages/satysfi-class-mdbook-satysfi-doc/satysfi-class-mdbook-satysfi.0.5.0/opam:1]: Can you fix the directory name?

```
packages/satysfi-class-mdbook-satysfi-doc/satysfi-class-mdbook-satysfi.0.5.0/opam

to

packages/satysfi-class-mdbook-satysfi-doc/satysfi-class-mdbook-satysfi-doc.0.5.0/opam
```

**Comment by puripuri2100** [comment]
Sorry late.
I fixed the reviewed point.

**Comment by puripuri2100** [comment]
Thank you! I fixed it.

**Comment by na4zagin3** [comment]
Thank you for waiting so long.  Let me retriger the CI by reopening the PR.

**Comment by na4zagin3** [comment]
Thanks!

**Pull request merged**